ci-operator/jobs/openshift/hypershift/openshift-hypershift-main-presubmits.yaml

```diff
@@ -598,6 +598,85 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-kubevirt-aws-ovn,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
```suggestion
    always_run: false
```

Reply:
I have add `always_run: false` to the config but the job is nog generated with false.